scrapyrt/conf/default_settings.py

```diff
@@ -6,6 +6,7 @@
 
 # Path to server log file
 LOG_FILE = None
+SPIDER_LOGFILE_TIMEFORMAT = '%Y-%m-%dT%H%M%S.%f'
```

Comment:
LOG_FILE is server file. SPIDER_LOG_FILE is file where spider logs will be placed. So I think keeping SPIDER prefix is ok, but yeah I added comment and changed LOGFILE to LOG_FILE.